libxrdp/xrdp_rdp.c

```diff
@@ -160,9 +161,43 @@ xrdp_rdp_read_config(struct xrdp_client_info *client_info)
                 client_info->use_fast_path = 0;
             }
         }
-        else if (g_strcasecmp(item, "disableSSLv3") == 0)
+        else if (g_strcasecmp(item, "ssl_protocols") == 0)
         {
-            client_info->disableSSLv3 = g_text2bool(value);
+            char s[strlen(value) + 2];
+            snprintf(s, sizeof(s), "%s%s", value, "x");
```

Comment:
Since you are copying the whole string anyway, you can add another space in the beginning and search for strings with spaces on both sides. That would guarantee that only whole words would match, not substrings.

But to be more user friendly, is would be better to check every word and report those that are unknown to xrdp.